src/scf/self_consistent_field.jl

```diff
@@ -102,7 +105,7 @@ Solve the Kohn-Sham equations with a SCF algorithm, starting at ρ.
         # Diagonalize `ham` to get the new state
         nextstate = next_density(ham; n_bands, ψ, eigensolver,
                                  miniter=1, tol=determine_diagtol(info),
-                                 n_ep_extra)
+                                 n_ep_extra, response.occupation_threshold)
```

Comment:
Didn't we decide to put it as a self_consistent_field kwarg and not in response?